c2corg_api/views/document.py

```diff
@@ -30,3 +31,64 @@ def _create_new_version(self, document):
         DBSession.add(meta_data)
         DBSession.add_all(versions)
         DBSession.flush()
+
+    def _update_version(self, document, comment, update_type, changed_langs):
+        assert update_type != UpdateType.NONE
+
+        meta_data = HistoryMetaData(comment=comment)
+        archive = self._get_document_archive(document, update_type)
+
+        cultures = \
+            self._get_cultures_to_update(document, update_type, changed_langs)
+        locale_versions = []
+        for culture in cultures:
+            locale = document.get_locale(culture)
+            locale_archive = self._get_locale_archive(locale, changed_langs)
+
+            version = DocumentVersion(
+                document_id=document.document_id,
+                culture=locale.culture,
+                version=999,
```

Comment:
I have made a few tests in my instance:
```
# select * from guidebook.documents_versions;
 id | document_id | culture | version | document_archive_id | document_locales_archive_id | history_metadata_id 
----+-------------+---------+---------+---------------------+-----------------------------+---------------------
  1 |           1 | fr      |       1 |                   1 |                           1 |                   1
  2 |           3 | fr      |       1 |                   2 |                           2 |                   2
  3 |           1 | fr      |     999 |                   3 |                           3 |                   3
```
Do we really need this "version" field?
In v5, it is used to display the version number in the history list:
http://www.camptocamp.org/routes/history/53806/fr
or to access an old version:
http://www.camptocamp.org/routes/53806/fr/14
or to diff 2 versions:
http://www.camptocamp.org/routes/diff/53806/fr/16/14

In wikipedia there is no version number displayed in the history:
https://fr.wikipedia.org/w/index.php?title=GitHub&action=history
An older version is accessed a kind of id:
https://fr.wikipedia.org/w/index.php?title=GitHub&oldid=116321058

@gberaudo suggests to simply use the table's id field (since auto-incremented and unique).